extensions/blocks/map/map.php

```diff
@@ -14,6 +14,47 @@
 	)
 );
 
+/**
+ * Fetch the A8C Mapbox access token from the WPCOM API.
+ *
+ * @return string
+ */
+function jetpack_fetch_mapbox_a8c_access_token() {
+	if ( ! Jetpack::is_active() ) {
+		return '';
+	}
+
+	$site_id = Jetpack_Options::get_option( 'id' );
+
+	$response = wp_remote_get( 'https://public-api.wordpress.com/wpcom/v2/sites/' . $site_id . '/mapbox' );
```

Comment:
Maybe we should store this in a transient for a little bit to avoid making a new request on every page load?